lib/runCall.js

```diff
@@ -71,7 +71,7 @@ module.exports = function (opts, cb) {
       stateManager.getAccount(createdAddress, function (err, account) {
         toAccount = account
         const NONCE_OFFSET = 1
-        toAccount.nonce = new BN(toAccount.nonce).addn(NONCE_OFFSET).toBuffer()
+        toAccount.nonce = new BN(toAccount.nonce).addn(NONCE_OFFSET).toArrayLike(Buffer)
```

Comment:
Actually, hold on a second. We need to define the endianess and size of the nonce field. Probably `toArrayLike(Buffer, 'be')` is the correct one, because RLP will remove the zeroes anyway.